curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-89-roman-numerals.md

```diff
@@ -18,52 +29,89 @@ For example, it would appear that there are at least six ways of writing the num
   VVIIIIII<br>
   XIIIIII<br>
   VVVI<br>
-  XVI<br>
+  XVI<br><br>
 </div>
 
 However, according to the rules only XIIIIII and XVI are valid, and the last example is considered to be the most efficient, as it uses the least number of numerals.
 
-The array, `roman`, contains one thousand numbers written in valid, but not necessarily minimal, Roman numerals; see [About... Roman Numerals](https://projecteuler.net/about=roman_numerals) for the definitive rules for this problem.
+The array, `roman`, will contain numbers written in valid, but not necessarily minimal, Roman numerals.
```

Comment:
```suggestion
The array, `roman`, will contain numbers written invalid, but not necessarily minimal, Roman numerals.
```

Reply:
It's supposed to be array with valid numerals, so invalid definitely cannot be there. Replace it with something else? *Correctly*? Or leave as it is?

Reply:
Maybe "written with valid"